### PR TITLE
Add resource requirement for running examples

### PR DESCRIPTION
**Pull request by xuqinziyue**
Signed-off-by: Qinziyue Xu <xqzy1014@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,3 +89,9 @@ make
  * [wavedrom](https://wavedrom.com/)
    Tool for generating waveform / timing diagrams.
 
+## Resource requirement
```
Comment (mithro) [inline README.md]: Could you format this as;

```md
## Resource Requirements

To run examples provided, please make sure these resources are available:
 * Memory: 5.5G
  * Disk space: 20G
```

Thank you!

**Comment by mithro** [comment]
@xuqinziyue ping?

**Comment by xuqinziyue** [comment]
@mithro Hi! Sorry haven't checked message for a while. I will commit the fix today

**Comment by mithro** [comment]
@xuqinziyue Can you please squash your two commits together and force push to your current branch?

**Comment by xuqinziyue** [comment]
Thanks! Sorry for the delay!

**Pull request merged**